blobs/howto/python/blob-devguide-py/blob-devguide-blobs.py

```diff
@@ -0,0 +1,410 @@
+        # Release a lease on a blob
+        lease_client.release()
+    # </Snippet_release_blob_lease>
+
+    # <Snippet_break_blob_lease>
+    def break_blob_lease(self, lease_client: BlobLeaseClient):
+        # Break a lease on a blob
+        lease_client.break_lease()
+    # </Snippet_break_blob_lease>
+
+    # <Snippet_get_blob_properties>
+    def get_properties(self, blob_service_client: BlobServiceClient, container_name):
+        blob_client = blob_service_client.get_blob_client(container=container_name, blob="sample-blob.txt")
+
+        properties = blob_client.get_blob_properties()
+
+        print(f"Blob type: {properties.blob_type}")
+        print(f"Blob size: {properties.size}")
+        print(f"Content type: {properties.content_settings.content_type}")
+        print(f"Content language: {properties.content_settings.content_language}")
+    # </Snippet_set_blob_properties>
+
+    # <Snippet_set_blob_properties>
+    def set_properties(self, blob_service_client: BlobServiceClient, container_name):
+        blob_client = blob_service_client.get_blob_client(container=container_name, blob="sample-blob.txt")
+
+        # Get the existing blob properties
+        properties = blob_client.get_blob_properties()
+
+        # Set the content_type and content_language headers, and populate the remaining headers from the existing properties
+        blob_headers = ContentSettings(content_type="text/plain",
+                                       content_encoding=properties.content_settings.content_encoding,
+                                       content_language="en-US",
+                                       content_disposition=properties.content_settings.content_disposition,
+                                       cache_control=properties.content_settings.cache_control,
+                                       content_md5=properties.content_settings.content_md5)
+        
+        blob_client.set_http_headers(blob_headers)
+    # </Snippet_set_blob_properties>
+
+    # <Snippet_set_blob_metadata>
+    def set_metadata(self, blob_service_client: BlobServiceClient, container_name):
+        blob_client = blob_service_client.get_blob_client(container=container_name, blob="sample-blob.txt")
+
+        # Retrieve existing metadata, if desired
+        metadata = dict(blob_client.get_blob_properties().metadata)
+
+        more_metadata = {'docType': 'text', 'docCategory': 'reference'}
+        metadata.update(more_metadata)
+
+        # Set metadata on the blob
+        blob_client.set_blob_metadata(metadata=metadata)
+    # </Snippet_set_blob_metadata>
+
+    # <Snippet_get_blob_metadata>
+    def get_metadata(self, blob_service_client: BlobServiceClient, container_name):
+        blob_client = blob_service_client.get_blob_client(container=container_name, blob="sample-blob.txt")
+
+        # Retrieve existing metadata, if desired
+        metadata = dict(blob_client.get_blob_properties().metadata)
```

Comment:
nit: not sure you need to wrap `blob_client.get_blob_properties().metadata` with the `dict` constructor since it already should be a dictionary.